### ISSUE TITLE
interference with (bash) tab-completion, double underscores and Vim

### ISSUE THREAD

**Issue by nielsbom** [wting/autojump#140]
First of all: thanks for autojump!

I installed autojump
I've got a 

```
__init__.py
```

file.
I want to open it using Vim so I type

```
$ vim __
```

I press Tab to autocomplete the name, the first time it gives no response, the
second Tab gives me autojump's suggestions.

It does _not_ occur when I use just one underscore:

```
$ vim _
```

and then Tab.

Nor does it occur with other commands like cd or nano.

```
$ nano __
```

and then Tab gives

```
$ nano __init__.py
```

as it should.

After thinking about this I realized the specific interference between
autojump and vim could hardly be by accident. So I RTFC'd and found:
https://github.com/joelthelion/autojump/blob/master/bin/autojump.bash#L30

So basically this is expected behaviour. However I can see this being a problem
for some.

What could be a solution is to be able to configure which characters are
used to "denote" the autojump directories. Like in an .autojumprc or something.
https://github.com/joelthelion/autojump/blob/master/bin/autojump.bash#L17

My shell-fu is lacking so I can't fix this myself, sorry!
